lib/smack/ConstantBVOps.cpp

```diff
@@ -31,7 +31,7 @@ unsigned getOpWidth(const Value *t) {
 }
 
 bool ConstantBVOps::runOnFunction(Function &f) {
-  if (!(SmackOptions::BitPrecise || SmackOptions::BitPrecisePointers))
+  if (SmackOptions::BitPrecise || SmackOptions::BitPrecisePointers)
```

Comment:
I think we don't need to guard against `BitPrecisePointers` since these are integer operations.